src/services/freelancer/freelancerAPI.ts

```diff
@@ -0,0 +1,35 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { RootState } from "store/store";
+import { IFreelancerSaved } from "../../components/FreelancerCard/FreelancerCard";
+import { REACT_APP_FREELANCER_PROFILE_INFO_URL } from "../../utils/consts/breakepointConsts";
+
+interface FreelancerId {
+  id: number;
+}
+
+const freelancerApi = createApi({
```

Comment:
Please don't use create Api anymore, follow the convention of the others below. Otherwise the user won't be logged out once the refresh token expires
```JSX
import apiSlice from "services/api/apiSlice";

const apiSliceWithTags = apiSlice.enhanceEndpoints({
  addTagTypes: ["Freelancers"],
});

const getTalentApi = apiSliceWithTags.injectEndpoints({
endpoints: (builder) => ({
    filter: builder.query<IFreelancer[], IFilters>({
      query: ({ keyWords, category, skills }) => ({
        url: `${FILTER_FREELANCER}/${keyWords}/${category}/${skills}`,
        method: "GET",
      }),
      providesTags: () => ["Freelancers"],
    }),
  }),
});
```